examples/with-react-esi/.babelrc

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This babelrc can be completely removed. Note that it's a bad practice to not use `next/babel`.
`next/babel` includes transform-runtime, preset-env, preset-react and a bunch of other configurations (object spread etc)